samples/vuejs/typescript/00 Custom Validator/src/pages/recipe/edit/pageContainer.tsx

```diff
@@ -0,0 +1,97 @@
+import Vue, {ComponentOptions} from 'vue';
```

Comment:
Yes, I know. And I simplify the sample as possible. There is a library [vuex](https://github.com/vuejs/vuex) to manage state like redux.